packages/dx-react-scheduler/docs/guides/views.md

```diff
@@ -0,0 +1,39 @@
+# React Scheduler - Views
+
+Views arrange appointments in different layouts. The Scheduler provides several view types. You can choose a single view type that presents your data in the best way or allow users to switch between several view types.
+
+## Related Plugins
+
+The following plugins render particular views:
+
+- [DayView](../reference/day-view.md) - renders a day view
+- [WeekView](../reference/week-view.md) - renders a week view
+- [MonthView](../reference/month-view.md) - renders a month view
+
+## Day View
+
+The `DayView` displays appointments for a specific date.
+
+The Scheduler arranges appointments from top to bottom. If their time intervals overlap, their width is decreased and they are placed next to each other.
```

Comment:
"their width is decreased and they are placed next to each other" -> their widths decrease and they are placed next to each other